namespaces/docker.yml

```diff
@@ -9,3 +9,103 @@ namespace:
     type: string
     description: >
       Docker container id
+  - name: event
+    type: group
+    description: >
+      Event emitted by docker daemon that describes some activity that occurred in the past
+    fields:
+    - name: action
+      type: string
+      example: create
+      description: >
+        What activity occurred
+    - name: status
+      type: string
+      example: create
+      description: >
+        What is the result of the activity
+    - name: type
+      type: string
+      example: container
+      description: >
+        Container or image or volume or network or daemon or plugin or service or node or secret or config
+    - name: time
+      type: date
+      example: 2017-08-22T14:47:29.146379+00:00
+      description: >
+        Date of the occurrence of the event
+    - name: container
+      type: group
+      description: >
+        What resource the activity targeted
+      fields:
+      - name: name
+        type: string
+        example: my-fedora
+        description: >
+          Name of the container
+      - name: image
+        type: string
+        example: registry.fedoraproject.org/fedora:latest
+        description: >
+          Image from which the container is derived
+      - name: state
+        type: group
+        description: >
+          State of the container
+        fields:
+        - name: Status
+          type: string
+          example: created
+          description: >
+            Result of the acticity performed on the container
+        - name: Running
```

Comment:
Is it the case that a container can be in a `Running` state at the same time it is `Paused`, `Restarting`, `OOMKilled`, or `Dead`?  If all five of these fields can be `true` at the same time, then separate fields seem warranted.  But if they can't, it might be better to consider using an enumeration.